zipline/finance/cancel_policy.py

```diff
@@ -0,0 +1,70 @@
+import abc
```

Comment:
I don't think we need to be updating every file with this:

http://www.apache.org/dev/apply-license.html#copy-per-file

> DO I HAVE TO HAVE A COPY OF THE LICENSE IN EACH SOURCE FILE?
Only one full copy of the license is needed per distribution. See the policy.